go/vt/sqlparser/ast_format_fast.go

```diff
@@ -19,7 +19,6 @@ package sqlparser
 
 import (
 	"fmt"
-
```

Comment:
The import formatting might complain about this line removal? We should fix the generation though to create a format that `goimports` thinks is ok.

Reply:
Ah yes, this is not correct and I'm not sure why it changed. The `static` linter in CI is complaining about it.

Reply:
@vmg I don't think this is something you changed, but an already existing long standing problem with the code generation for this file. I thought it might be easy to fix while you're in here already.